test/planned_subvol_test.rb

```diff
@@ -207,4 +208,122 @@
     end
   end
 
+  context ".read_from_xml" do
```

Comment:
Once again. Reading of subvolumes from control_xml is already unit-tested at https://github.com/yast/yast-storage-ng/blob/master/test/proposal_settings_test.rb#L269  I think we can remove this whole context. It makes no much sense to test exactly the same twice (just with different wording).